src/index.html

```diff
@@ -9,10 +13,73 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
The link to the stylesheet should be a CSS file, not a SCSS file. The browser doesn't understand SCSS syntax.

Reply:
In my task I should use exactly scss